lib/broadway_cloud_pub_sub/pull_client.ex

```diff
@@ -36,10 +36,23 @@ defmodule BroadwayCloudPubSub.PullClient do
   def receive_messages(demand, ack_builder, config) do
     max_messages = min(demand, config.max_number_of_messages)
 
-    config
-    |> execute(:pull, %{"maxMessages" => max_messages})
-    |> handle_response(:receive_messages)
-    |> wrap_received_messages(ack_builder)
+    :telemetry.span(
+      [:broadway_cloud_pub_sub, :pull_client, :pull_request],
+      %{
+        requested_messages: max_messages,
+        total_demand: demand,
+        topology_name: config.broadway[:name]
+      },
+      fn ->
+        result =
+          config
+          |> execute(:pull, %{"maxMessages" => max_messages})
+          |> handle_response(:receive_messages)
+          |> wrap_received_messages(ack_builder)
+
+        {result, %{topology_name: config.broadway[:name]}}
```

Comment:
Typically the stop metadata should be a superset of the start one. :)

Reply:
Sure, will add

Reply:
done